ddl/partition.go

```diff
@@ -1335,6 +1335,8 @@ func checkExchangePartitionRecordValidation(w *worker, pt *model.TableInfo, inde
 		} else if len(pi.Columns) == 1 {
 			sql = buildCheckSQLForRangeColumnsPartition(pi, index, schemaName, tableName)
 		}
+	case model.PartitionTypeList:
```

Comment:
well, list partition can be `list` or `list columns` please refer to `PartitionTypeRange`

Reply:
OK, I have added `list partition`, PTAL.